modules/project/variables.tf

```diff
@@ -302,10 +302,12 @@ variable "shared_vpc_service_config" {
   description = "Configures this project as a Shared VPC service project (mutually exclusive with shared_vpc_host_config)."
   # the list of valid service identities is in service-agents.yaml
   type = object({
-    host_project                = string
-    service_identity_iam        = optional(map(list(string)), {})
-    service_identity_subnet_iam = optional(map(list(string)), {})
-    service_iam_grants          = optional(list(string), [])
+    host_project                 = string
+    host_project_iam             = optional(list(string), [])
```

Comment:
I would restrict to only assign compute.networkUser, and rename to host_network_users

Reply:
let's just rename to

- `network_users`
- `network_subnet_users`

Reply:
Updated variables and code!